### ISSUE TITLE
Primary Key when data type is Binary

### ISSUE THREAD

**Issue by Pakalex** [balderdashy/sails-mysql#215]
Using sails, set a data type as binary:

id: {
  type: 'binary',
  primaryKey: true,
  size: 16
},

The schema is generating with a varchar primary key.

I added this custom line to solve the problem.

if(type === 'BLOB')
{
  return attrName + 'BINARY(16) NOT NULL PRIMARY KEY'; 
}

Location:
File: sql.js
schema: function
Line number: 82

**Comment by devinivy** [comment]
Thanks for catching that– would love it if you could submit a PR!

**Comment by Pakalex** [comment]
You're welcome, i would rather that you do it, because i'm new at this.

**Comment by devinivy** [comment]
I posted a PR for this: #217

**Comment by sailsbot** [comment]
Thanks for posting, @Pakalex.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
